clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -168,6 +168,9 @@ public class VenicePushJob implements AutoCloseable {
   public static final String SEND_CONTROL_MESSAGES_DIRECTLY = "send.control.messages.directly";
   public static final String SOURCE_ETL = "source.etl";
   public static final String ETL_VALUE_SCHEMA_TRANSFORMATION = "etl.value.schema.transformation";
+  public static final String SCHEMA_READER_ENABLED = "schema.reader.enabled";
+  public static final String SCHEMA_SYSTEM_STORE_D2_ZK_HOST = "schema.system.store.d2.zk.host";
+  public static final String SCHEMA_SYSTEM_STORE_D2_SERVICE_NAME = "schema.system.store.d2.service.name";
```

Comment:
Instead of having d2 zk host differentiated by feature, let's use something region based. VPJ already has `d2.zk.hosts.<region name>` configs. So, we can have a config that defines the parent region, or we could reuse `source.grid.fabric`

Reply:
It is a good idea to reuse existing configs. But neither `d2.zk.hosts.<parent.controller.region.name>` or `d2.zk.hosts.<source.grid.fabric>` works for the following case. Please correct me if I'm wrong.

When we use prod-lor1 data to repush the store, the push job will consume prod-lor1 kakfa topic, which may have records with new KME so we need to retrieve KME from prod-lor1 routers. But the push job will run in WAR, whose `source.grid.fabric` is prod-lva1. Using `d2.zk.hosts.prod-lva1` will connect to prod-lva1 routers and may not be able to retrieve the new KME as KME is not registered in all fabrics at the same time.

Reply:
In the latest change of the PR, we do not need to specify d2 zk hosts or service name before running repush job. Those information will be retrieved from controller by `getRepushInfo` api.